ts/rabbit.ts

```diff
@@ -43,9 +45,15 @@ export default class Rabbit extends EventEmitter {
       return;
     }
     this.connecting = true;
-    let connection = await amqp.connect(this.url, this.socketOptions);
-    let channel = await this.createChannel(connection);
-    await this.initChannel(channel);
+    this.connection = await amqp.connect(this.url, this.socketOptions);
```

Comment:
Is this the `consume` connection? If so why don't we name it?

Reply:
Only not to break someone if it was using connection. I will change it and ship it with a major update. Will also remove log4js dependency in a new pr.